src/main/groovy/edu/wpi/first/gradlerio/test/JavaTestPlugin.groovy

```diff
@@ -48,7 +48,8 @@ class JavaTestPlugin implements Plugin<Project> {
 
                 if (OperatingSystem.current().isUnix()) {
                     env["LD_LIBRARY_PATH"] = ldpath
-                    env["DYLD_FALLBACK_LIBRARY_PATH"] = ldpath // On Mac it isn't 'safe' to override the non-fallback version.
```

Comment:
Isn't the comment here the exact reason for _not_ doing this change?

Reply:
The comment is wrong for 10.15 and above. Any version below 10.15 will ignore DYLD_LIBRARY_PATH so it is safe.

Reply:
If you've verified that this is safe and works, then no problem 👍